app/core/vueRouter.js

```diff
@@ -26,6 +26,17 @@ export default function getVueRouter () {
           component: () => import(/* webpackChunkName: "ParentsView" */ 'app/views/landing-pages/parents/PageParents'),
           props: (route) => ({ showPremium: true, type: route.query.type })
         },
+        {
+          path: '/parents-v2',
+          component: () => import(/* webpackChunkName: "ParentsView" */ 'app/views/landing-pages/parents-v2/PageParents'),
+          props: (route) => ({ showPremium: true, type: route.query.type }),
```

Comment:
do we use this query parameters in new version?

Reply:
The new component is inherited from the old one, so we _might_ use it.